DocumentDataSubmission/src/main/resources/wsdl/NhinXDR.wsdl

```diff
@@ -0,0 +1,138 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This wsdl file is for an XDS.b Document Repository Actor -->
+<definitions xmlns:soap="http://schemas.xmlsoap.org/wsdl/soap12/"
+             xmlns:tns="urn:ihe:iti:xdr:2007"
```

Comment:
We need only NhinXDR2.0 wsdl. The reason we have 2 wsdls for XDR is we had 2 specifications for xdr. For DDS we will use 2.0 only.